.gitignore

```diff
@@ -17,3 +17,9 @@ src/client/static/data/
 src/client/static/_redirects
 src/client/static/sitemap.xml
 src/client/static/sw.js
+
```

Comment:
We don't add these to a project's gitignore. You should create a global gitignore on your machine. This is how: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

If you would want to make a contribution to an open source project for example, they don't want every PR to also include gitignore changes specific to that contributor. So that's why we also don't put them in the project's gitignore.